src/index.ts

```diff
@@ -73,6 +76,8 @@ export function autoRetry(
                 typeof result.parameters?.retry_after === 'number' &&
                 result.parameters.retry_after <= maxDelay
             ) {
+                const timeString = new Date().toLocaleString();
+                debug(`[${timeString}] Retrying ${method} after ${result.parameters.retry_after} seconds`);
```

Comment:
`debug` already adds timing information. Also, "retrying" doesn't make it clear that the call hasn't happened yet.
```suggestion
                debug(`Will retry ${method} after ${result.parameters.retry_after} seconds`);
```